services/github/src/handlers/index.js

```diff
@@ -79,15 +81,6 @@ class Handlers {
     assert(!this.deprecatedInitialStatusPq, 'Cannot setup twice!');
     assert(!this.rerunPq, 'Cannot setup twice!');
 
-    // This is a powerful Queue client without scopes to use throughout the handlers for things
```

Comment:
this is moved to the `main.js` so we can inject different fake mocks during testing